cornichon-core/src/main/scala/com/github/agourlay/cornichon/core/Session.scala

```diff
@@ -75,7 +75,7 @@ case class Session(private val content: Map[String, Vector[String]]) extends Any
       }
   }
 
-  def addValues(tuples: Seq[(String, String)]) = tuples.foldLeft(this)((s, t) ⇒ s.addValue(t._1, t._2))
+  def addValues(tuples: (String, String)*) = tuples.seq.foldLeft(this)((s, t) ⇒ s.addValue(t._1, t._2))
```

Comment:
i guess thats the interesting question. that was the point where i thought you might have explicitly chosen to use a Seq :D
http://www.scala-lang.org/old/node/5579 says it does allocate a collection but it does only copy the object references of the passed args into it. This post is talking about scala 2.8 though.